tools/nix/parts/all.nix

```diff
@@ -22,8 +22,11 @@
 
       shellHook = ''
         ${config.devShells.rust.shellHook or ""}
-        export HOME=$PWD/.home
         export DYLD_FALLBACK_LIBRARY_PATH=$(rustc --print sysroot)/lib
+
+        export HOME=$PWD/.home
+        export PATH=$PWD/.home/.cargo/bin:$PATH
+        cargo install tauri-cli --version "^2.0.0-alpha"
```

Comment:
Later today, I'll contribute an actual Nix derivation for this CLI, so please omit this line and the one prepending PATH prior to merging. 

Generally `shellHook` should not be used to make any permanent modifications to an end user's system, as it violates the principle of least surprise. In a Nix environment there's typically a more idiomatic way to accomplish what's needed.

Reply:
I'm resolving the conversation. Please open a PR when this one is merged